backend/src/main/java/gov/cdc/usds/simplereport/service/UploadService.java

```diff
@@ -78,7 +90,7 @@ public String processPersonCSV(InputStream csvStream) throws IllegalGraphqlArgum
             rowNumber++;
             try {
                 _ps.addPatient(
-                    parseUUID(row.get(FACILITY_ID)),
+                    parseUUID(row.get(FACILITY_ID)),    // todo: verify this is a facility in the current Org.
```

Comment:
This is potentially a big mistake if the spreadsheet has the wrong `facilityId`.

Reply:
This is already handled by `updatePersonFacility` which is called from `addPatient`

Reply:
What I mean by this is that the facilityId guid comes from the spreadsheet.